File: BreakingTheRecords.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

class Result {
    public static List<Integer> breakingRecords(List<Integer> scores) {
        int minRecordBreaks = 0;
        int maxRecordBreaks = 0;
        
        int minScore = scores.get(0);
        int maxScore = scores.get(0);
        
        for (int i = 1, n = scores.size(); i < n; i++)
        {
            int score = scores.get(i);
            
            if (score > maxScore)
            {
                maxScore = score;
                maxRecordBreaks++;
            }
            
            if (score < minScore)
            {
                minScore = score;
                minRecordBreaks++;
            }
        }
        
        return Arrays.asList(maxRecordBreaks, minRecordBreaks);
    }
}

public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

        int n = Integer.parseInt(bufferedReader.readLine().trim());

        String[] scoresTemp = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");

        List<Integer> scores = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            int scoresItem = Integer.parseInt(scoresTemp[i]);
            scores.add(scoresItem);
        }

        List<Integer> result = Result.breakingRecords(scores);

        for (int i = 0; i < result.size(); i++) {
            bufferedWriter.write(String.valueOf(result.get(i)));

            if (i != result.size() - 1) {
                bufferedWriter.write(" ");
            }
        }

        bufferedWriter.newLine();

        bufferedReader.close();
        bufferedWriter.close();
    }
}

File: BreakingTheRecords.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'breakingRecords' function below.
#
# The function is expected to return an INTEGER_ARRAY.
# The function accepts INTEGER_ARRAY scores as parameter.
#

def breakingRecords(scores):
    result = [0, 0]
    
    minScore = scores[0]
    maxScore = scores[0]
    
    for score in scores[1:]:
        if score > maxScore:
            maxScore = score
            result[0] += 1
        elif score < minScore:
            minScore = score
            result[1] += 1
    
    return result

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    n = int(input().strip())

    scores = list(map(int, input().rstrip().split()))

    result = breakingRecords(scores)

    fptr.write(' '.join(map(str, result)))
    fptr.write('\n')

    fptr.close()
